### PR TITLE
Implement support for pgx's CopyFrom

### PR DESCRIPTION
**Pull request by Jille**
I implemented my own feature request https://github.com/kyleconroy/sqlc/issues/1350.

### CODE REVIEW

**Comment by Jille** [comment]
Any thoughts on this?

File: internal/codegen/golang/templates/pgx/queryCode.tmpl [hunk 1]
```diff
@@ -112,6 +114,53 @@ func (q *Queries) {{.MethodName}}(ctx context.Context, {{.Arg.Pair}}) (pgconn.Co
...
+{{end -}}
+{{- if $.EmitMethodsWithDBArgument}}
+func (q *Queries) {{.MethodName}}(ctx context.Context, db DBTX, {{.Arg.SlicePair}}) (int64, error) {
+	return db.CopyFrom(ctx, {{.TableIdentifier}}, {{.Arg.ColumnNames}}, &iteratorFor{{.MethodName}}{rows: {{.Arg.Name}}})
+{{- else}}
+func (q *Queries) {{.MethodName}}(ctx context.Context, {{.Arg.SlicePair}}) (int64, error) {
+	return q.db.CopyFrom(ctx, {{.TableIdentifier}}, {{.Arg.ColumnNames}}, &iteratorFor{{.MethodName}}{rows: {{.Arg.Name}}})
+{{- end}}
+}
+{{end}}
```
Comment (kyleconroy) [inline internal/codegen/golang/templates/pgx/queryCode.tmpl:162]: Let's put this code into a different file, something called `{query}_copyfrom.go`

**Review by kyleconroy — CHANGES_REQUESTED** [review]
* Please rename `:copyFrom` to `:copyfrom`.
* We need to return an error if you try to use `:copyfrom` using stdlib PostgreSQL or MySQL
* The newly generated code should live in a different file

Let me know if you have any questions.

**Comment by kyleconroy** [comment]
> Did you want a separate file for each query?

Up to you! Probably easier to make it one file for each package.

> What about settings to override the filename for those?

That can come later in a follow-on PR.

> And what about the MyQueryParams and MyQueryRow structs?

We should keep those where they are today.

Hope that help!

**Comment by Jille** [comment]
Thanks!

All done. Let me know when you want me to rebase+squash.

**Review by kyleconroy — APPROVED** [review]

**Pull request merged**